.github/workflows/formatting-and-tests.yml

```diff
@@ -0,0 +1,46 @@
+name: Formatting & Tests
+
+on:
+    - pull_request
```

Comment:
The other repos we have:

```
    pull_request:
        types: [opened, synchronize, reopened, ready_for_review]
```
But I'm not actually sure why 😁 maybe add for consistency?

Reply:
I guess it just filters out draft PRs